include/oneapi/dpl/internal/dynamic_selection_impl/dynamic_load_policy.h

```diff
@@ -14,6 +14,8 @@
 #include <memory>
 #include <ostream>
 #include <vector>
+#include <exception>
```

Comment:
Why does this file have #include <exception> and the rest don't ?

Reply:
Do you mean #include <exception>?

Reply:
Do you mean #include exception? Only dynamic load and auto tune use it.